src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -1,18 +1,17 @@
 package seedu.address.logic.commands;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CLASSID;
```

Comment:
Could consider splitting CLASSID into two words? i.e. `PREFIX_CLASS_ID`

Similarly in other locations and for STUDENTID as well.

Reply:
Noted. I will work on it!

Reply:
Done with this!